### DISCUSSION TITLE
Anyone using the DFP-34X-2C2 with Vivo Fibra Sao Paulo?

### DISCUSSION THREAD

**Discussion by phtoth** [Anime4000/RTL960x#241] [General]
Hello everyone, how are you doing?

Has anyone configured the DFP-34X-2C2 with the Vivo Fibra provider from São Paulo?

I am trying everything, and getting the fake O5, but no Internet at all.

I tried the configs at the doc, but no success

Best regards

**Comment by Tj639** [comment]
> Olá a todos, como vão?
> 
> Alguém já configurou o DFP-34X-2C2 com a operadora Vivo Fibra de São Paulo?
> 
> Estou tentando de tudo e conseguindo o O5 falso, mas não tenho internet nenhuma.
> 
> Tentei as configurações no documento, mas sem sucesso
> 
> Atenciosamente

Olá amigo , faz tempo está pergunta , vc conseguiu ?

**Reply by phtoth** [reply]
Infelizmente Não, porém não parei mais para ver isso também
